portal-api/src/v1/controllers/utilisation-report-service/fee-record-corrections/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './pending-corrections.controller';
 export * from './fee-record-correction-transient-form-data.controller';
 export * from './get-fee-record-correction.controller';
+export * from './get-fee-record-correction-review.controller';
```

Comment:
I think most of our controller files in portal-api don't have 'get' or 'put' or whatever action preceding them, so I think this filename should just be:
```suggestion
export * from './fee-record-correction-review.controller';
```